charterafrica-dev.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.8"
```

Comment:
Any reason this file is not named `docker-compose`?

Reply:
@kilemensi A similar file would maybe be needed when migrating codeforafrica

Reply:
Then we'll have docker-compose.codeforafrica.yml, docker-compose.charterafrica.yml, etc. i.e. Similar to how we've named Dockerfiles.

PS: Since it's all about services, we can have a single docker-compose file with all services and a dev can choose which app they want to run e.g. `docker compose up codeforafrica`, `docker compose up charterafrica`, etc.

cc @thepsalmist if what I'm saying makes sense

Reply:
Yes makes sense to me @kilemensi a single docker-compose file, since the only dependency is MongoDB, the two apps can both depend on that. So we'll have two separate DBs within the clusture, and pass the app specific URL e.g `mongodb://127.0.0.1:27017/charterafrica` and `mongodb://127.0.0.1:27017/codeforafrica`